### PR TITLE
Update graph/tile based switches for AIE_profile/trace_settings

### PR DESCRIPTION
**Pull request by IshitaGhosh**
#### Problem solved by the commit
The new switches under AIE_profile/trace_settings used keywords that are not authorized to be disclosed to user. So, they needed to be changed to align with marketing request. Other switches are also updated to be more specific.
Also, the related warning/error messages were updated to remove use of unauthorized keyword.
A few other improvements were also added.

### CODE REVIEW

**Review by stsoe — APPROVED** [review]

**Comment by IshitaGhosh** [comment]
retest this please

**Comment by dayeh-xilinx** [comment]
retest this please. Windows build host fixed.

File: src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp [hunk 1]
```diff
@@ -617,7 +617,7 @@ bool AieTracePlugin::configureStartIteration(xaiefal::XAieMod& core)
     std::string metricsStr = xrt_core::config::get_aie_trace_metrics();
```
Comment (rozumx) [inline src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp:617]: Copyright date needs to be updated on line 2.

File: src/runtime_src/xdp/profile/plugin/aie_profile/aie_plugin.h [hunk 2]
```diff
@@ -62,9 +62,9 @@ namespace xdp {
     bool checkAieDevice(uint64_t deviceId, void* handle);
```
Comment (rozumx) [inline src/runtime_src/xdp/profile/plugin/aie_profile/aie_plugin.h:62]: Copyright date needs to be updated on line 2.

File: src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.h [hunk 3]
```diff
@@ -149,6 +149,11 @@ namespace xdp {
       EventVector memoryCounterResetEvents;
```
Comment (rozumx) [inline src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.h:149]: Copyright date needs to be updated on line 2.

**Review by rozumx — APPROVED** [review]
Approved with some minor issues (e.g., copyright dates need to be updated).

File: src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.h (hunk 3 above)
Comment (IshitaGhosh) [reply src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.h:150]: Done

File: src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp (hunk 1 above)
Comment (IshitaGhosh) [reply src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp:617]: Done

**Review by jvillarre — APPROVED** [review]

**Pull request merged**